dev-docs/bidders/glimpse.md

```diff
@@ -1,95 +1,117 @@
+deals_supported: true
 media_types: banner
+gvl_id: 1012
 ---
 
 ## Overview
 
 ```
-Module Name: Glimpse Protocol Adaptor
+Module Name: Glimpse Protocol Bid Adapter
 Module Type: Bidder Adapter
-Maintainer: tim@glimpseprotocol.io
+Maintainer: publisher@glimpseprotocol.io
 ```
 
 ## Description
 
 This module connects publishers to Glimpse Protocol's demand sources via Prebid.js. Our
-innovative marketplace protects consumer privacy while allowing precise targeting. It is
-compliant with GDPR, DPA and CCPA.
+innovative marketplace protects consumer privacy while allowing precise targeting.
 
-This module was built and tested against prebid 3.21.0 and so compatibility against
-version 2 and earlier is unknown.
+## Supported Media Types
 
-## Media Types
-
-| Type     | Support                                                            |
+| Type     | Sizes                                                              |
 | -------- | ------------------------------------------------------------------ |
-| `Banner` | Fully supported for 320x50, 300x250, 300x600, 728x90, and 970x250. |
+| `Banner` | 300x250, 300x600, 320x50, 728x90, 970x250                          |
 
-## Bid Parameters
+## Setup
 
-The only parameter is `placementId` and it is required.
+This section shows how to configure your Prebid.js integration to work with the Glimpse module.
 
-### Banner
+### Prerequisites
 
-| Name          | Scope    | Description                                                                                                      | Example                | Type   |
-| ------------- | -------- | ---------------------------------------------------------------------------------------------------------------- | ---------------------- | ------ |
-| `placementId` | Required | An identifier associated unique to a publisher and ad unit. Values can be obtained through our publisher portal. | 'glimpse-demo-300x250' | String |
+Before you start to configure Glimpse, you will need to build a `prebid.js` file with the Glimpse module included,
+and include both `gpt.js` and `prebid.js` in the `head` of each page with supply. An example of a typical pair of script tags might be:
 
-## Setup Guide
+```html
+<script
+  async
+  src="https://securepubads.g.doubleclick.net/tag/js/gpt.js"
+></script>
 
-Follow these steps to configure and add the glimpse module to your Prebid.js integration.
+<script
+  async
+  src="https://<PUBLISHER_DOMAIN>/prebid.js"
+></script>
+```
 
-### 0. Preconditions
+### Registering as a publisher
 
-- A built prebid module with the glimpse adaptor included
-- You've included the built prebid adaptor and GPT script in your websites html code
-- You've setup GAM mappings
+Coming soon.
 
-### 1. Create an account and setup your domain via the Publisher Portal
+### Registering your domain
 
 Coming soon.
 
-### 2. Enable Glimpse as a bidder on your ad units
+### Registering your ad units
+
+Coming soon.
+
+## Configuration
+
+### Bid Requests
```

Comment:
I'm confused. Why do you have two sets of params -- what's the difference between 'bid requests' here and 'bidder' below?

Also, the header levels are possible wrong here?

Reply:
We separated them to allow for different levels of scoping (e.g. to specify keywords that apply to individual units vs. keywords that apply to all units), plus it prevents publishers from having to specify the Glimpse account ID for each bid request.

Double-checked the header levels and they seem okay (the `configuration` section purposely doesn't fall under `setup`). I've pushed a change to remove the `banner` headers though.